.github/workflows/node.js.yml

```
@@ -9,12 +10,7 @@ on:
     branches: ["main"]
     paths-ignore:
     - "**.md"
-  merge_group:
-    paths-ignore:
-    - "LICENSE"
-    - "CODEOWNERS"
-    - "**.md"
```

```suggestion
    paths-ignore:
    - "LICENSE"
    - "CODEOWNERS"
    - "**.md"
```

We may still need this paths ignore added to the `push` and `pull_request` keys?